settings.job

```diff
@@ -0,0 +1,3 @@
+{
+    "schedule": "0 */15 * * * *"
```

Comment:
should be light enough for every 5 minutes imo?

Reply:
They currently run in ~20 seconds (excluding the time to spin up the node deps and build cache on travis) so yes they could!